akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -50,7 +50,8 @@ Next we'll discuss each of these in detail.
 
 ### Command handler
 
-The command handler is a function with 3 parameters for the `ActorContext`, current `State`, and `Command`.
+The command handler is a function with @java[2 parameters for]@scala[3 parameters for the `ActorContext`,]
+current `State`, and `Command`.
```

Comment:
Looks a little bit weird for Java with comma when there is only two listed parameters. Not a native speaker though so maybe that is correct.

Reply:
I think that is how to enumerate things in English (at least some dialects), but it's weird when having only two items. Here we have 2 or 3 at the moment, so removing the comma for simplicity.